packages/main/src/plugin/extensions-updater/extensions-updater.ts

```diff
@@ -21,7 +21,7 @@ import { app } from 'electron';
 import { coerce, satisfies } from 'semver';
```

Comment:
Should extension-updater be also moved ?

Reply:
updater and catalog will be follow-up